pyradiosky/skymodel.py

```diff
@@ -1662,6 +1655,8 @@ def from_recarray(
             spectral_index=spectral_index,
         )
 
+        # TODO: not sure if the following is really necessary if we're newly creating
+        # the full object
```

Comment:
This is needed because of recarrays. When they go away, we don't need to do this.

Reply:
OK. Removed the TODO.